streaming/README.md

```diff
@@ -236,6 +236,44 @@ When TLS/SSL is enabled on the slave, the default behavior will be to not connec
     ssl skip certificate verification = yes
 ```
 
+#### Trusted Certificate
```

Comment:
This needs some grammar fixes @joelhans but, more importantly, it's not the master certificate that we need to bring to the slave, since it gets it from the master already. 

What we do here  is specify the locations of Certificate Authority (CA) certificates, necessary to verify the master certificate. In other words, we configure the slave so that it can validate the certificate chain. 

The parameters in the config file should be called 
SSL_CTX_load_verify_locations CApath
SSL_CTX_load_verify_locations CAfile

If we do that, we don't need to explain much about what goes in those two parameters, just add a link to 
 https://www.openssl.org/docs/man1.0.2/man3/SSL_CTX_load_verify_locations.html

Reply:
I changed the variable names in the stream.conf and I brought to the documentation the latest version of the OpenSSL link instead the previous that you gave us.
With the changes in the text that I executed, I hope I could remove the ideat that the option allows we use only the Netdata master certificate, the correct is to say that it is possible to add the certificate to the Trusted list and instead to search in the whole list of Trusted certificates, we can work directly with the netdata master.
In the last PR I also changed the plugin_d.c, because when the Netdata is compiled without SSL it was giving a warning. Considering that this was happening due the previous PR that we merged to fix the other SSL problems, I changed the variable here, because it has relationship with the stream and SSL.